tests/test.rs

```diff
@@ -710,11 +710,13 @@ fn test_plaintext_pass_wrong_pass() {
 }
 
 #[test]
+#[cfg_attr(not(feature = "rust-crypto"), ignore)]
```

Comment:
I've chosen to use `ignore` here so that the tests are still compiled even when the feature is disabled, hopefully avoiding accidental compile breakages.

I based it on some ignored tests (for unix-socket).

Is there a reason you believe we should prefer disabling them?